### PR TITLE
Build fixes.

### PR DESCRIPTION
**Pull request by ry**
Fixes @Pato91's [Linux/release issue](https://github.com/denoland/deno/issues/438#issuecomment-410939516).

Removes the redundant `//third_party/v8/third_party/depot_tools`. People may have to run setup.py after this commit.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /out/
 *.pyc
 gclient_config.py_entries
+Cargo.lock
```
Comment (piscisaureus) [inline .gitignore]: Does this have any effect? IIRC this file is in third_party.

File: .gn [hunk 2]
```diff
@@ -23,6 +23,10 @@ default_args = {
   use_ozone = false
   use_udev = false
 
+  # TODO(ry) We may want to turn on CFI at some point. Disabling for simplicity
+  # for now. See http://clang.llvm.org/docs/ControlFlowIntegrity.html
```
Comment (piscisaureus) [inline .gn]: Why is this necessary? "Simplicity" seems like a good argument but I don't see what gets simpler, I see only an added flag that begs for explanation.

File: gclient_config.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
         'v8/tools/swarming_client': None,
         'v8/third_party/instrumented_libraries': None,
         'v8/third_party/android_tools': None,
+        'v8/third_party/depot_tools': None,
```
Comment (piscisaureus) [inline gclient_config.py]: this was bugging me. Good to see it gone.

File: .gitignore (hunk 1 above)
Comment (ry) [reply .gitignore]: I believe it got created for me. I think Cargo is run in the root.

File: .gn (hunk 2 above)
Comment (ry) [reply .gn]: It fixes the linux release build (which isn't tested in CI) 
https://github.com/denoland/deno/issues/438#issuecomment-410939516
